app/models/audit/event/check.rb

```diff
@@ -6,19 +6,19 @@ class Check
       def initialize(
         user:,
         client_ip:,
-        resource:,
+        resource_id:,
         privilege:,
-        role:,
+        role_id:,
         operation:,
         success:
       )
         @user = user
         @client_ip = client_ip
-        @resource = resource
         @privilege = privilege
-        @role = role
         @operation = operation
         @success = success
+        @resource_id = resource_id
+        @role_id = role_id
```

Comment:
Can you please verify that all calls to check.rb pass resource id and role id? Also from unit tests

Reply:
Done